.rubocop.yml

```diff
@@ -6,3 +6,8 @@ Documentation:
 
 Style/MultilineMethodCallIndentation:
   EnforcedStyle: indented
+
+Style/PercentLiteralDelimiters:
+  PreferredDelimiters:
+    '%i': []
```

Comment:
Codeclimate has `()` as default delimiters, which is different from ruby style guide.